src/client/elm/Explorer/Request.elm

```diff
@@ -93,6 +94,7 @@ type alias UpdateQueues =
     , foundationAccount : UpdateQueue T.AccountAddress
     , electionDifficulty : UpdateQueue Float
     , euroPerEnergy : UpdateQueue Relation
+    , bakerStakeThreshold : UpdateQueue T.Amount
     }
 
```

Comment:
The UpdateQueues have also been slightly updated, now instead of authorization there are level1 and level2 key updates that are there